github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml

```diff
@@ -131,7 +131,7 @@ presubmits:
       annotations:
         fork-per-release: "true"
       always_run: true
-      optional: true
+      optional: false
       decorate: true
       skip_report: false
```

Comment:
Do you want to remove skip_report line ?
because the default value is false (right?)

Reply:
that's not the point of this PR

Reply:
np
/lgtm